### ISSUE TITLE
New release 1.4.5 caused Anaconda shortcut not working.

### ISSUE THREAD

**Issue by timeregained** [ContinuumIO/menuinst#45]
After upgrade to menuinst 1.4.5, all shortcuts of anaconda(Anaconda Navigator, IPython,...etc) cannot work, I guess it caused by new win32.py but I cannot figure out which parts of it.

After downgrading it to 1.4.4, all shortcuts can work normally.

Below is my conda information.

Current conda install:

               platform : win-64
          conda version : 4.3.18
       conda is private : False
      conda-env version : 4.3.18
    conda-build version : 2.1.13
         python version : 3.6.1.final.0
       requests version : 2.14.2
       root environment : D:\Apps\x64\Continuum\Miniconda3  (writable)
    default environment : D:\Apps\x64\Continuum\Miniconda3
       envs directories : D:\Apps\x64\Continuum\Miniconda3\envs
                          C:\Users\jti\AppData\Local\conda\conda\envs
                          C:\Users\jti\.conda\envs
          package cache : D:\Apps\x64\Continuum\Miniconda3\pkgs
                          C:\Users\xxx\AppData\Local\conda\conda\pkgs
           channel URLs : https://conda.anaconda.org/anaconda-fusion/win-64
                          https://conda.anaconda.org/anaconda-fusion/noarch
                          https://repo.continuum.io/pkgs/free/win-64
                          https://repo.continuum.io/pkgs/free/noarch
                          https://repo.continuum.io/pkgs/r/win-64
                          https://repo.continuum.io/pkgs/r/noarch
                          https://repo.continuum.io/pkgs/pro/win-64
                          https://repo.continuum.io/pkgs/pro/noarch
                          https://repo.continuum.io/pkgs/msys2/win-64
                          https://repo.continuum.io/pkgs/msys2/noarch
                          https://conda.anaconda.org/intel/win-64
                          https://conda.anaconda.org/intel/noarch
            config file : C:\Users\xxx\.condarc
           offline mode : False
             user-agent : conda/4.3.18 requests/2.14.2 CPython/3.6.1 Windows/10 Windows/10.0.15063
          administrator : unknown

**Comment by mingwandroid** [comment]
Ah sorry about that. I will make a new release very soon. In the meantime, if you open cwp.py and change the end to the following:

```
documents_folder, exception = get_folder_path(FOLDERID.Documents)
if exception:
    documents_folder, exception = get_folder_path(FOLDERID.PublicDocuments)
if not exception:
    os.chdir(documents_folder)
```

.. you should be able to launch things again.

**Comment by mingwandroid** [comment]
This is fixed now with release 1.4.6.
